frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -594,31 +594,28 @@ function UsageTab({ featureFlag }: { id: string; featureFlag: FeatureFlagType })
                 <b>Log</b>
                 <div className="text-muted">{`Feature flag calls for "${featureFlagKey}" will appear here`}</div>
             </div>
-            <EventsTable
-                fixedFilters={{
-                    event_filter: '$feature_flag_called',
-                    properties: propertyFilter,
-                }}
-                fixedColumns={[
-                    {
-                        title: 'Value',
-                        key: 'value',
-                        render: function renderEventProperty(_, { event }: EventsTableRowItem) {
-                            return event?.properties['$feature_flag_response']?.toString()
-                        },
+            <Query
+                query={{
+                    kind: NodeKind.DataTableNode,
+                    source: {
+                        kind: NodeKind.EventsQuery,
+                        select: [
+                            ...defaultDataTableColumns(NodeKind.EventsQuery),
+                            featureFlag.filters.multivariate
+                                ? 'properties.$feature_flag_response'
+                                : "if(properties.$feature_flag_response == 1, 'true', 'false')",
+                        ],
+                        event: '$feature_flag_called',
+                        properties: propertyFilter,
                     },
-                ]}
-                startingColumns={['person']}
-                fetchMonths={1}
-                pageKey={`feature-flag-` + featureFlagKey}
-                showPersonColumn={true}
-                showEventFilter={false}
-                showPropertyFilter={false}
-                showCustomizeColumns={false}
-                showAutoload={false}
-                showExport={false}
-                showActionsButton={false}
-                emptyPrompt={`No events received`}
+                    full: true,
+                    showEventFilter: false,
+                    showPropertyFilter: false,
+                    showColumnConfigurator: false,
+                    showExport: false,
+                    showActions: false,
+                    showReload: false,
```

Comment:
Flyby: I'm not sure of all of the context here, though I'm sure there will be other `showWhatever` default-true fields in the future. Perhaps it's better to omit `full: true` and explicitly only show the elements you care about? 🤔

Reply:
yeah fair!